hera_cal/redcal.py

```diff
@@ -157,6 +160,9 @@ def filter_reds(reds, bls=None, ex_bls=None, ants=None, ex_ants=None, ubls=None,
             which must be specified.
         max_bl_cut: cut redundant groups with average baselines lengths longer than this. Same units as antpos
             which must be specified.
+        max_dims: maximum number of dimensions required to specify antenna positions (up to some arbitary shear).
```

Comment:
Explain what dimensions mean in a comment. Are these unspecified degrees of freedom that are left undetermined after redcal that need to be fixed by an abscal model (or may not even be abscalable?)

Reply:
Thanks for clarifying here about subarrays.